### ISSUE TITLE
Absence of interest in peers

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7322]
**Preferred term label:**
Absence of interest in peers

**Synonyms**

**Definition (free text, please give PubMed ID)**
When a child shows little or no signs of interest in similar age peers. 
DSM-V (need to look at ISBN used from other terms)

**Parent term (use hpo.jax.org/app)**
Abnormal social behavior
HP:0012433

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708
